src/doc/en/installation/meson.rst

```diff
@@ -106,18 +104,29 @@ Installing is done with the following command:
 
     $ meson install -C builddir
 
-This will then install in the directory specified by ``--prefix``, e.g.
-``build-install/lib/python3.11/site-packages/sage``.
-Usually, this directory is not on your Python path, so you have to use:
+This will install the project to currently active Python environment, 
+or to the system Python environment if no environment is active.
+When editable install is used, it is not necessary to reinstall after each compilation.
 
-.. CODE-BLOCK:: shell-session
+.. NOTE::
 
-    $ PYTHONPATH=build-install/lib/python3.11/site-packages ./sage
+    If you want to install the project to a different directory, you can specify
+    the ``--prefix`` option when running the ``meson setup`` command:
 
-When editable install is used, it is not necessary to reinstall after each compilation.
+    .. CODE-BLOCK:: shell-session
+
+        $ meson setup builddir --prefix=/desired/install/path -Dpython.install_env=prefix
+
+    This will then install in the directory specified by ``--prefix``,
+    in particular the root folder will be be installed to
+    ``/desired/install/path/lib/python3.11/site-packages/sage``.
+    Usually, this directory is not on your Python path, so you have to use:
+
+    .. CODE-BLOCK:: shell-session
+
+        $ PYTHONPATH=/desired/install/path ./sage
```

Comment:
are you sure this actually works, in the previous version the `PYTHONPATH` is set to `…/site-packages` (I haven't tested either)